pages/common/json5.md

```diff
@@ -0,0 +1,7 @@
+# json5
+
+> A command-line tool for transpiling JSON5 files to JSON.
```

Comment:
Any reason you are using the word "transpile" ? You are just converting files from one format to another. It's not even source code.

Reply:
I guess it's more just converting. :+1: I'll change that.